app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,10 @@
+class FoodsController < ApplicationController
+    def index
+      @ingredient = params[:q]
+      @food_count = FoodsService.search_ingredients(@ingredient)[:totalHits]
+      # binding.pry
+      @foods = FoodsService.food_list(@ingredient) 
+      # binding.pry
+
+    end
```

Comment:
Ideally, if we can have one object getting sent to the view, we should try to do that. The one object will contain all data necessary for displaying in the view. You mentioned in a comment that you would Ideally call a Facade here and then let your method in the Facade call your Service, and I would agree with that refactor. We shouldn't have to do any manipulation/formatting of our data in the controller.